### PR TITLE
add rkt to glossary

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: content/en/docs/reference/glossary/rkt.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka: 
+tags:
+- security
+- tool
+---
+ A security-minded, standards-based container engine.
+
+<!--more--> 
+
+rkt is an application {% glossary_tooltip term_id="container" %} engine featuring a {% glossary_tooltip term_id="pod" %}-native approach, a pluggable execution environment, and a well-defined surface area. rkt allows users to apply different configurations  at both the pod and application level and each pod executes directly in the classic Unix process model, in a self-contained, isolated environment.
```
Comment (tfogo) [inline content/en/docs/reference/glossary/rkt.md]: The shortcodes should be in the form `{{< glossary_tooltip text="pod" term_id="pod" >}}`. Can you make that change? Apart from that, lgtm.

**Comment by Rajakavitha1** [comment]
@tfogo Please take a look at the updated changes.

**Comment by zparnold** [comment]
@xmulligan Were you able to make the changes @tfogo suggested?

**Comment by xmulligan** [comment]
@zparnold yes I was able to add the text labels that @tfogo suggested

**Comment by sftim** [comment]
I think a good title for this PR would be “add rkt to glossary”

**Pull request merged**